### ISSUE TITLE
 NetlifyCMS problem finding image on disk

### ISSUE THREAD

**Issue by pluralsigher** [Vagr9K/gatsby-advanced-starter#215]
I get this error when I try to create a new blog post with an uploaded image, I uploaded the image within the admin page. 
 "Field "image" must not have a selection since type "String" has no subfields."
 
 The image is placed in the folder "images" (themes/advanced/content/images) next to the blog posts (themes/advanced/content) as is done in the repo. 
 
 In the .mdx file the cover is set to: 
cover: images/imageName.jpg
(no qoutes)
 
 These links might contain a solution, but I have not been able to solve it yet. 
 https://github.com/gatsbyjs/gatsby/issues/13469
 https://theleakycauldronblog.com/blog/problems-with-gatsby-image-and-their-workarounds

**Comment by vasfvitor** [comment]
Hi, I'm a beginner but I'll try to help. 
Are you using the Netlify config that comes with this starter or did you created one from scratch?

The only thing I can think of is to compare the frontmatter in the .mdx files you created via admin page and the default post. You can send it here so I can take a look.

From what I've read around here the theme content folder is supposed to be only for development purposes (check discussion #154). But I believe it's supposed to work regardless, it's just a recommendation.

**Comment by pluralsigher** [comment]
Hi @vasfvitor 

Thank you, Im also a beginner

I got it to work, but I have to manually change the image path.

I duplicated the article so you can see the one that comes from NetlifyCMS and the one that I edited in the same photo. 
(The duplicated article has an extra dot (' . ') that I added by mistake, ignore that.). 

The GraphQL query or Netlify also adds an extra .md for some reason...

But the main problem is that when I publish the article with Netlify the image is placed in `themes/advanced/content/images/amonarciaespanola.jpg `when it should be placed in `content/assets/amonarciaespanola.jpg`. 

I changed the `contentDir` and `assetsDir`, thank you for the tip!, you can see the settings i use in the screenshot. 

Do you know how I can change the path where the image is stored from Netlify? 

Thanks once again 
<img width="1792" alt="Screen Shot 2021-11-23 at 13 44 03" src="https://user-images.githubusercontent.com/94419591/143096556-5d1dd9d6-fa23-43e8-a6f8-f819a13e3e50.png">

**Comment by vasfvitor** [comment]
As far as I remember, you don't have to put the .md (if you are doing it) in the files when creating, Netlify CMS alredy does it. If it's automatic adding .md two times, I don't know why.

About the configuration: It's in a file called `config.yml` in the folder `themes/advanced/static/admin` but I'd recommend copy it over /static/admin and use it there and edit the folder paths to match your configuration. At discussion #162 I was still configuring and figuring stuff out, but may have something to add to your situation.

Also take a look at [Netlify CMS docs](https://www.netlifycms.org/) there's some basics that you might be interest in.

> I got it to work, but I have to manually change the image path.

Yeah, I did that too until I finally got it right without editing.
